site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -347,6 +347,11 @@ const workers: DNode[] = [
 ];
 ```
 
+Notice that we have added an `id` property to each child.
+This is needed so that we can tell the children apart.
```

Comment:
"This is needed so that we can tell the children apart." reads a bit odd, perhaps: This is needed to differentiate between the children.  ?